test/scale_optional_test.cpp

```diff
@@ -16,7 +16,7 @@ using scale::EncodeError;
 using scale::ScaleDecoderStream;
 using scale::ScaleEncoderStream;
 
-// TODO(yuraz): PRE-119 refactor to parameterized tests
+// TODO(Harrm): PRE-119 refactor to parameterized tests
```

Comment:
Issue label looks weird

Reply:
Oh, it's from the good ol' times Kagome wasn't Kagome, but was simply Polkadot Runtime Environment (PRE)...